src/core/public/chrome/chrome_service.tsx

```diff
@@ -158,6 +161,29 @@ export class ChromeService {
         }
         this.collapsibleNavHeaderRender = render;
       },
+      registerNavLink: (useCase: ChromeUseCase, navLink: ChromeRegistrationNavLink) => {
```

Comment:
The function name `registerNavLink` doesn't sound appropriate, seems what we are doing here is register a relation between the app and the use case, but not register a specific nav link.

Reply:
Changed to `addNavToGroup`.